cluster-provision/images/vm-image-builder/fedora-with-test-tooling/cloud-config

```diff
@@ -2,13 +2,46 @@
 password: fedora
 chpasswd: { expire: False }
 ssh_pwauth: yes
+write_files:
+    - path: /etc/modules-load.d/mlx5.conf
+      content: |
+        mlx5_core
+        mlx5_ib
+    - path: /etc/modules-load.d/mlx4.conf
+      content: |
+        mlx4_core
+        mlx4_ib
+    - path: /etc/modules-load.d/i40e.conf
+      content: |
+        i40e
+    - path: /etc/modules-load.d/igb.conf
+      content: |
+        igb
+    - path: /etc/systemd/system/wait-for-cloud-init.service
+      owner: root:root
+      permissions: '0755'
+      content: |
+          [Unit]
+          Description=Wait for cloud-init to complete
+          Before=qemu-guest-agent.service
+          After=cloud-final.service
+
+          [Service]
+          Type=oneshot
+          ExecStartPre=cloud-init status --wait
```

Comment:
btw this means the cloud-init is finished, but maybe with errors right ?

Reply:
According to my experience `cloud-final` will not complete then, so it will not even reach this command. Here an example of failing tests. The guest agent here never starts because the user-data is bad and `cloud-final` never succeeds.

This may not be the most convenient behaviour with my unit, but at least we see right now clearly a failing test.

Reply:
Forgot to add the link: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/6108/pull-kubevirt-e2e-k8s-1.19-sig-network/1419925101213650944

Reply:
maybe worth to update kubevirt "Retried login attempt after two minutes failed" to say something like (did cloud-init finish ?)